tox.ini

```diff
@@ -210,133 +212,81 @@ setenv=
-  rgw_multisite_container: PLAYBOOK = site-docker.yml.sample
-  shrink_mon_container: PLAYBOOK = site-docker.yml.sample
-  shrink_mon_container: MON_TO_KILL = mon2
-  shrink_osd_container: PLAYBOOK = site-docker.yml.sample
-  shrink_osd_container: COPY_ADMIN_KEY = True
-  shrink_osd: COPY_ADMIN_KEY = True
-  container_podman: PLAYBOOK = site-docker.yml.sample
+  centos: CEPH_ANSIBLE_VAGRANT_BOX = centos/7
+  fedora: CEPH_ANSIBLE_VAGRANT_BOX = fedora/29-atomic-host
+  # Set the vagrant box image to use
+  centos-non_container: CEPH_ANSIBLE_VAGRANT_BOX = centos/7
+  centos-container: CEPH_ANSIBLE_VAGRANT_BOX = centos/atomic-host
+  podman: CEPH_ANSIBLE_VAGRANT_BOX = fedora/29-atomic-host
+  ubuntu: CEPH_ANSIBLE_VAGRANT_BOX = ceph/ubuntu-xenial
+
+  # Set the ansible inventory host file to be used according to which distrib we are running on
+  ubuntu: _INVENTORY = hosts-ubuntu
+  INVENTORY = {env:_INVENTORY:hosts}
+  container: CONTAINER_DIR = /container
+  container: PLAYBOOK = site-docker.yml.sample
   storage_inventory: COPY_ADMIN_KEY = True
-  storage_inventory_container: PLAYBOOK = site-docker.yml.sample
+  podman: PLAYBOOK = site-docker.yml.sample
+  non_container: PLAYBOOK = site.yml.sample
+  container-purge_cluster: PURGE_PLAYBOOK = purge-docker-cluster.yml
+  shrink_mon: MON_TO_KILL = mon2
+  shrink_osd: COPY_ADMIN_KEY = True
 
   rhcs: CEPH_STABLE_RELEASE = luminous
-  luminous: CEPH_STABLE_RELEASE = luminous
-  luminous: CEPH_DOCKER_IMAGE_TAG = latest-luminous
-  luminous: CEPH_DOCKER_IMAGE_TAG_BIS = latest-bis-luminous
-  luminous: UPDATE_CEPH_STABLE_RELEASE = luminous
-  luminous: UPDATE_CEPH_DOCKER_IMAGE_TAG = latest
-  mimic: CEPH_STABLE_RELEASE = mimic
-  mimic: CEPH_DOCKER_IMAGE_TAG = latest-mimic
-  mimic: CEPH_DOCKER_IMAGE_TAG_BIS = latest-bis
-  mimic: UPDATE_CEPH_STABLE_RELEASE = mimic
-  mimic: UPDATE_CEPH_DOCKER_IMAGE_TAG = latest
   lvm_osds: CEPH_STABLE_RELEASE = luminous
-  lvm_osds: PLAYBOOK = site.yml.sample
-  lvm_osds_container: CEPH_STABLE_RELEASE = luminous
-  lvm_osds_container: PLAYBOOK = site-docker.yml.sample
-  bluestore_lvm_osds: CEPH_STABLE_RELEASE = luminous
-  bluestore_lvm_osds_container: CEPH_STABLE_RELEASE = luminous
-  bluestore_lvm_osds_container: PLAYBOOK = site-docker.yml.sample
-  lvm_batch_container: PLAYBOOK = site-docker.yml.sample
-  lvm_batch_container: CEPH_STABLE_RELEASE = luminous
-  update_cluster: ROLLING_UPDATE = True
-  update_docker_cluster: ROLLING_UPDATE = True
-  add_osds: CEPH_STABLE_RELEASE = luminous
-  add_osds_container: CEPH_STABLE_RELEASE = luminous
-  rgw_multisite: CEPH_STABLE_RELEASE = luminous
-  rgw_multisite_container: CEPH_STABLE_RELEASE = luminous
+
+  rhcs: CEPH_STABLE_RELEASE = luminous
+  dev: CEPH_DOCKER_IMAGE_TAG = latest-master
+  dev: CEPH_DOCKER_IMAGE_TAG_BIS = latest-master-bis
+  dev: UPDATE_CEPH_DOCKER_IMAGE_TAG = latest-master
+  dev: UPDATE_CEPH_DEV_BRANCH = master
+  dev: UPDATE_CPEH_DEV_SHA1 = latest
+  dev: CEPH_STABLE_RELEASE = mimic
```

Comment:
what's the purpose of setting 'CEPH_STABLE_RELEASE' for 'dev' environment? I thought 'dev' environment always install latest master packages of Ceph?

Reply:
@ajarr it's for the upgrade scenario, you come from a stable release and upgrade to 'dev'